tutorial/compiler.md

```diff
@@ -46,4 +46,4 @@ In addition, for each accelerator (mpi/openmp/openacc/python) that you will use
 
 Python is considered to be an accelerator and must additionally specify shared\_suffix.
 
-The default compilers can provide examples compatible with your system once pyccel has been executed at least. These json files can be found in the folder pyccel/compilers/
+The default compilers can provide examples compatible with your system once pyccel has been executed at least. To export the json file describing your setup, use the `--export-compile-info` flag or the `export_compile_info` argument and provide a target file name.
```

Comment:
Can you please provide an example of how this feature should be used in practice?

For instance, this does not work:
```bash
$ pyccel --export-compile-info comp.json
error: the following arguments are required: N
```

Reply:
I didn't change the command significantly so we still need to be pyccelising something. Ie:
```
pyccel --export-compile-info comp.json my_file.py
```
Now I think about it, this seems like a bad idea. I will change it so that it works with the command you describe above. Do you still think I should give an example if it is fixed?

Reply:
Fixed 2a548fc2

Reply:
Thanks for fixing this, it is much better now. Should we maybe enforce the `.json` extension of the preference file? That is, add `.json` to the file name if the extension is missing, and raise an error if the file name has the wrong extension.

Reply:
Fixed 18e37e13